services/s3/src/main/resources/codegen-resources/customization.config

```diff
@@ -16,6 +27,16 @@
         }
       ]
     },
+    "UploadPartRequest": {
+      "inject": [
+        {
+          "PartType": {
+            "shape": "PartType",
+            "documentation": "Enum flag that indicates whether the request is a last part or not. Will be DEFAULT or LAST. This field will be set by the Encryption team as part of their logic, the SDK does not set the field. This field is needed since encryption of the parts must happen sequentially, and for AES-GCM (the only supported encryption algorithm) it must know which part is last in order to write the auth tag to the end of the ciphertext object"
```

Comment:
lol, we probably wouldn't want to add so much detail here. Suggesting:
```
Indicates whether the request is a last part or not. This field will not be included in the request payload.

<p>
It is usually not required outside of testing or advanced libraries, so most users should not need to configure it.
```

Reply:
updated documentation